src/env/Makefile.mk

```diff
@@ -29,25 +29,32 @@ endif BUILD_FC_BINDING
 
 if INSTALL_MPICXX
 bin_SCRIPTS += src/env/mpicxx
+if BUILD_ABI_LIB
+bin_SCRIPTS += src/env/mpicxx_abi
+endif BUILD_ABI_LIB
 endif INSTALL_MPICXX
 
 # create a local copy of the compiler wrapper that will actually be installed
 if BUILD_BASH_SCRIPTS
 src/env/mpicc: $(top_builddir)/src/env/mpicc.bash
 	cp -p $? $@
-src/env/mpicc_abi: $(top_builddir)/src/env/mpicc_abi.bash
+src/env/mpicc_abi: $(top_builddir)/src/env/mpicc.bash
```

Comment:
Alternatively, follow the example of `mpif77`:
```
src/env/mpif77: src/env/mpifort
        ln -f -s mpifort $@
```
We could replace link with cp if that is preferred.

Reply:
OK, a symlink is definitely a better approach. I'll update the PR later and ping you back.